sphinx/writers/latex.py

```diff
@@ -1555,9 +1555,10 @@ def depart_definition_list_item(self, node):
     def visit_term(self, node):
         # type: (nodes.Node) -> None
         self.in_term += 1
-        ctx = '}] \\leavevmode'  # type: unicode
-        if node.get('ids'):
-            ctx += self.hypertarget(node['ids'][0])
+        ctx = ''  # type: unicode
+        for node_id in node['ids']:
+            ctx += self.hypertarget(node_id, anchor=False)
+        ctx += '}] \\leavevmode'  # type: unicode
```

Comment:
After this change, `\label` goes inside `\item`.  I believe `\phantomsection` is not needed in this case.